cmd/genji/shell/command.go

```diff
@@ -366,3 +378,118 @@ func runDumpCmd(db *genji.DB, tables []string, w io.Writer) error {
 	_, err = fmt.Fprintln(w, "COMMIT;")
 	return err
 }
+
+// runSaveCommand saves the currently opened database in the given filename.
+// If a file already exists, it will be overwritten.
+func runSaveCmd(ctx context.Context, db *genji.DB, engineName string, path string) error {
```

Comment:
`ctx` because we'll open a new db along the way, which will then share the context with the runner as intended (signal handling notably as I assumed that the `db` will stop if `ctx` is closed).